File: CPP/Product.cpp
class Product
{
    private:
        int Price;
        int IdProduct;

    public:

        Product(){}

        //GetSet untuk Price
        void setPrice(int Price)
        {
            this->Price = Price;
        }

        int getPrice()
        {
            return this->Price;
        }

        //GetSet untuk IdProduct
        void setIdPRoduct(int IdProduct)
        {
            this->IdProduct = IdProduct;
        }

        int getIdProduct()
        {
            return this->IdProduct;
        }

        ~Product(){}
    
};

File: Java/Product.java
package W2.Latihan1.Java;

class Product {
    private int Price;
    private String IdProduct;

    public Product(int Price, String IdProduct) {
        this.setPrice(Price);
        this.setIdProduct(IdProduct);
    }

    public Product() {
    }

    // GetSet IdProduct
    public String getIdProduct() {
        return IdProduct;
    }

    public void setIdProduct(String IdProduct) {
        this.IdProduct = IdProduct;
    }

    // GetSet Price
    public int getPrice() {
        return Price;
    }

    public void setPrice(int Price) {
        this.Price = Price;
    }

    public void printProduct() {
        System.out.println("Price      : " + this.Price);
        System.out.println("Id Product : " + this.IdProduct);
    }

}
